metadata/nz_building_outlines.xml

```diff
@@ -104,48 +104,17 @@
-
-**Accuracy Specification:**
-The Building Outlines contain spatial detail reflective of the visible characteristics of building outlines as seen from the source imagery. The building outlines were created by an automated method of feature extraction. 
+        <gco:CharacterString>
+This dataset consists of polygon outlines representing buildings within mainland New Zealand. A building outline is a 2D representation of the roof outline of a building which has been classified from LINZ aerial imagery using a combination of automated and manual processes to extract and orthogonalise a building roof outline. Building roof outlines observed in aerial imagery larger than or equal to 10 square meters are captured in this dataset. Each building polygon represents a building roof outline and this may include structures such as garages and large sheds.
 
-**Guidelines for Building Outline Extraction:**
-**The following rules were used for capturing buildings;**
-• Buildings under construction are not captured.
-Standalone dwellings:
-• Primary building structures are captured as separate polygons. Adjoining buildings are captured as a single polygon.
-• Extensions, sunrooms, balconies, patios, and annexes are captured as part of the primary building structure.
-• Permanent buildings structures such as sheds and greenhouses, not attached to the primary building structure, but larger than 10m² are captured as individual buildings.
-• Inner courtyards within buildings are captured as a hole within the building polygon if resolution of imagery allows. These holes should be no smaller than 10m².
-Townhouses:
-• Adjoining townhouses are captured a single polygon.
-Commercial:
-• Adjoining commercial buildings are captured as separate polygons with coincident shared edges.
-Rural/farmland areas:
-• Hay bales and silage are not captured.
-• Water tanks are captured as buildings. Minimum size is 16.5m².
-General:
-• Non-permanent objects such as shipping containers, caravans, large vehicles and trampolines should not be captured as buildings.
+This dataset represents the most recent set of building outlines which have been determined from the most recent LINZ aerial imagery available in each region of the country and available on the LINZ Data Service. Current coverage includes areas in Northland, Auckland, Waikato, Hawke’s Bay, Manawatu-Whanganui, Wellington, Tasman, West Coast, Marlborough, Canterbury, Southland and Otago. As new aerial imagery becomes available, this dataset will be updated with new building outlines derived from this new imagery. Since aerial imagery is flown only every few years in each region of the country, building outlines only reflect those buildings observed at the date of this aerial imagery capture, and may not reflect buildings constructed after this time period.  
```

Comment:
I suggest adding a sub-heading here to break the content of the abstract up:

**Data vintage** 
This dataset shows the most recent set of building outlines extracted from the most recent LINZ aerial imagery available on the LINZ Data Service. 

Current coverage includes areas in Northland, Auckland, Waikato, Hawke’s Bay, Manawatu-Whanganui, Wellington, Tasman, West Coast, Marlborough, Canterbury, Southland and Otago. This dataset will be updated and expanded as new aerial imagery becomes available. 

Since aerial imagery is flown only every few years in each region of the country, building outlines only reflect those buildings observed at the date of this aerial imagery capture, and may not reflect buildings constructed after this time period.